### ISSUE TITLE
Try to upgrade angular 8.1 to angular 9.

### ISSUE THREAD

**Issue by whitegoat** [angular/angular#35332]
I try to upgrade angular 8.1 to angular 9.

# 🐞 Bug report

- [ yes] build
- [yes ] serve

### Is this a regression?

 Yes, the previous version in which this bug was not present was: ....

### Description

In tsconfig.json have added the below line
"angularCompilerOptions": {
    "enableIvy": true
  }

Try to build or server getting the error.
 
105 export declare class TreeModule {
                         ~~~~~~~~~~
node_modules/angular2-multiselect-dropdown/multiselect.component.d.ts:85:22 - error NG6002: Appears in the NgModule.imports of SessionModule, but could not be resolved to an NgModule class

**Comment by jeripeierSBB** [comment]
might be solved by #35188 / #35227

**Comment by gkalpak** [comment]
Based on the error message, this is probably a duplicated of #35255.
Closing this one in favor of #35255. Let's track this issue there.
